### ISSUE TITLE
Create Expect.info

### ISSUE THREAD

**Issue by cdibbs** [alsatian-test/alsatian#488]
Hey, for v3.0, what about adding an info method:

```typescript
const tmpDir = await this.makeTmpDir();
Expect.info("tmpDir", tmpDir);
```

```
not ok 8 someDemoTestMethod_forExample
 ---
   message: "should strictly (===) equal"
   severity: fail
   data:
     got: onething
     expect: another
   info:
     tmpDir: /tmp/rx847gn89
 ...
```

**Comment by jamesadarich** [comment]
I think this is something similar to something that is being experimented with at the moment.

I have an `Expect.log` which outputs a list of strings passed to it on a failed test :)

Would this cover the requirement out is there something else you'd need?

**Comment by jamesadarich** [comment]
closed by #508
